src/main/java/ro/unibuc/triplea/domain/auth/service/GameService.java

```diff
@@ -0,0 +1,31 @@
+package ro.unibuc.triplea.domain.auth.service;
+
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.unibuc.triplea.domain.auth.repository.GameRepository;
+import ro.unibuc.triplea.application.auth.dto.response.GameResponse;
+
+@Service
+public class GameService {
+    
+    private final GameRepository gameRepository;
+
+    @Autowired
```

Comment:
Autowired can be avoided, also constructor. Use @RequiredArgsConstructor as used in many other files as injection

Reply:
fixed